apps/meteor/client/sidebar/header/CreateChannelWithData.js

```diff
@@ -86,7 +86,7 @@ const CreateChannelWithData = ({ onClose, teamId = '', reload }) => {
 		}
 
 		onClose();
-		reload();
+		reload && reload();
```

Comment:
```suggestion
		reload?.();
```

Let's use the chain syntax. 

Better to do either if (reload) reload(); OR, this ^ imo.

Reply:
Right. Thank you